### ISSUE TITLE
Update WIKI with technical details: FAQ

### ISSUE THREAD

**Issue by maulieshah** [hackforla/tdm-calculator#1866]
### Overview
We need to add technical details for how the FAQ page works so that the stakeholder's new vendor or in house development team can take over the project.

#### Details
This page also serves as an information source for development issues while the site is under development by Hack for LA.

### Action Items
- [x] Fill in Technical Details

### Resources/Instructions
- https://github.com/hackforla/tdm-calculator/wiki/FAQ

#### Slides
- [Staging Deck](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg)
   - [Staging Deck, slide 1866](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg/edit?slide=id.g34e3a8aaa05_0_0#slide=id.g34e3a8aaa05_0_0)
- Release Deck
   - [YYYY-MM-DD Release Deck, slide 1866]

**Comment by ExperimentsInHonesty** [comment]
Product needs to add additional screen shots for the editing mode of this page

**Comment by DorianDeptuch** [comment]
Availability: Weekday Evenings
ETA: 4/15 End of Day

**Comment by ryanfchase** [comment]
@DorianDeptuch -- I think you moved this to the "Done" column accidentally. I will move it back to in progress. Please also fill out your update for tonight, thanks!

<details><summary>Instructions</summary>

1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (if necessary): "Add any pictures that will help illustrate what you are working on."
</details> 

1. Progress: 
2. Blockers: 
3. Availability:
4. ETA:
5. Pictures (if necessary):

**Comment by ryanfchase** [comment]
@DorianDeptuch oh -- and if you have completed your work and it needs to be reviewed, please move to "Technical Review" and add `ready for dev lead - 311` so that I can have a look

**Comment by ryanfchase** [comment]
ETA for review: 2025-04-16 6pm PT (tonight)

**Comment by DorianDeptuch** [comment]
@ryanfchase Yes, thank you! I'll remember to add the `ready for dev lead - 311` tag from now on.

**Comment by ryanfchase** [comment]
This was reviewed with dev leads on 2025-04-16. Feedback from John Darragh:
- We won't be able to fully vet the details of this until someone familiarizes themselves with it and makes a complete review
- John is ok with leaving this info as a base, and then correcting details that might be wrong in the future.

Closing as complete.
